tools/selinux/icinga2.sh

```diff
@@ -67,6 +67,7 @@ sepolicy manpage -p . -d icinga2_t
 
 # Label the port 5665
 /sbin/semanage port -a -t icinga2_port_t -p tcp 5665
+/sbin/semanage port -a -t redis_port_t -p tcp 6380
```

Comment:
Isn't the policy about _our Redis_ whereas the name implies _the regular Redis_?

Reply:
Apart from naming: do we want to cover port 6379 (i.e. actual Redis default) as well?

Reply:
And – if yes – under which name?

Reply:
Yes, the goal is to be able to connect icinga2 with redis in the first place though, redis already has its own policy and we don't need to define such types by our own. However, by default this port is not included in this policy and so I just add the `6380` to the allowed ports of redis and allow Icinga2 to connect on all available ports of this type.

Reply:
While using an unassigned port is bad, it is probably too late to change the decision, so giving it the correct label for its use is the right thing to do. The shell script is only for development, so do not forget to change this in the corresponding package.